packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -216,9 +224,9 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     if (isStickyTop || isStickyBottom) {
       let height = 0,
         width = 0;
-      if (this.nonStickyContent) {
-        height = this.nonStickyContent.offsetHeight;
-        width = this.nonStickyContent.scrollWidth;
+      if (this.nonStickyContent && this.nonStickyContent.firstElementChild) {
+        height = (this.nonStickyContent.firstElementChild as HTMLElement).offsetHeight;
```

Comment:
Sorry for going back and forth on this.

@tharshada and I were discussing this today, and we believe that using offsetHeight of the firstElementChild may not be the right thing to do here. 

nonStickyContent can have a CSS applied to it, when it is sticky. In case the style applied changes the offsetHeight, that diff in height would not be considered when syncing stickyContentTop <-> stickyContentBottom <-> placeholder.

We tried to repro this locally by adding a style which has padding applied to it, and we were able to repro a broken behavior. In this case, the content was hidden behind the Sticky header/footer.

[Another issue - the current code in shouldComponent will always fail. stickyContentTop/stickyContentBottom is set with nonStickyContent offsetHeight and compared with offsetHeight of firstChildElement]